index.html

```diff
@@ -602,298 +609,143 @@ <h3>Ethereum EIP712 Signature 2021</h3>
            </pre>
       </section>
     </section>
-  </section>
 
-  <section>
-    <h2>Test Vectors</h2>
-    <p>
-      The following test vectors are provided to assist with implementers.
-    </p>
+    <section id="test-vectors">
+      <h2>Test Vectors</h2>
+      <p>
+        The following test vectors are provided to assist implementers. Some of the given test vectors specify
+        <code>inputOptions</code> which are options to be passed when creating a proof.
+        These can include options specifying the <code>domain</code>, <code>types</code>, <code>primaryType</code>,
+        <code>verificationMethod</code>, <code>date</code>, <code>embedAsURI</code>, and <code>embed</code>.
+      </p>
 
-    <p>
-      The following is an example ECDSA K-256 keypair that can be used to generate EIP712 signatures:
-    </p>
-    <pre class="example">
+      <p>
+        The following is an example Ethereum-compatible hexadecimal private key, and corresponding <code>did:pkh</code>
+        <code>verificationMethod</code> that can be used to assist with test vectors:
+      </p>
+      <pre class="example">
         {
-          "keypair_0": {
-            "id": "did:example:aaaabbbb#issuerKey-1",
-            "type": "EcdsaSecp256k1VerificationKey2019",
-            "controller": "did:example:aaaabbbb",
-            "publicKeyJwk": {
-              "kty": "EC",
-              "crv": "secp256k1",
-              "x": "cmbYyDC6cbm807_OmFNYP4CLEL0aB2F1UG683SxFkXM",
-              "y": "zBw5HAh0cJM4YimSQvtYM1HFhzUXVUgrDhxJ70aajt0",
-            },
-            "privateKeyJwk": {
-              "kty": "EC",
-              "crv": "secp256k1",
-              "x": "cmbYyDC6cbm807_OmFNYP4CLEL0aB2F1UG683SxFkXM",
-              "y": "zBw5HAh0cJM4YimSQvtYM1HFhzUXVUgrDhxJ70aajt0",
-              "d": "u7QuEl6W0XNppEY0iMVjATT99tC9acwV3Z2keEqvKGo"
-            }
-          }
-        }              
+          "privateKey": "0x149195a4059ac8cafe2d56fc612f613b6b18b9265a73143c9f6d7cfbbed76b7e",
+          "verificationMethod": "did:pkh:eth:0xAED7EA8035eEc47E657B34eF5D020c7005487443#blockchainAccountId"
```

Comment:
```suggestion
          "verificationMethod": "did:pkh:eip155:1:0xAED7EA8035eEc47E657B34eF5D020c7005487443#blockchainAccountId"
```
Committed in 580581afc1a2da02166525bbb32173a1bc2fc1fd in #43